packages/deployment/package.json

```diff
@@ -20,9 +20,9 @@
   "author": "Agoric",
   "license": "Apache-2.0",
   "dependencies": {
-    "@agoric/assert": "^0.3.15-dev.2",
-    "@agoric/install-ses": "^0.5.29-dev.2",
-    "@agoric/marshal": "^0.4.29-dev.2",
+    "@agoric/assert": "^0.3.15-dev.1",
+    "@agoric/install-ses": "^0.5.29-dev.1",
+    "@agoric/marshal": "^0.4.29-dev.1",
     "chalk": "^2.4.2",
```

Comment:
I don't think I understand this change well.

What prompted it? How did you diagnose that it was necessary?

Reply:
This was due to a mistake I made when testing.  I've rolled that all back now.